pf2-core/Core/Rules/Combat/MeleeAttack.cs

```diff
@@ -1,7 +1,8 @@
+using PF2.Core.Rules.Creature;
 using PF2.Core.Utility;
 using System.Collections.Generic;
 
-namespace PF2.Core.Rules
+namespace PF2.Core.Rules.Combat
```

Comment:
Not super important but I believe the CRB calls it "Encounter Mode" not "Combat" but Combat is intuitive enough.

Reply:
True! Maybe Rules.EncounterMode.Combat and also Rules.EncounterMode.Social which will allow for separation of Social interaction logic and combat logic.
This train of thought also would mean having a Rules.ExplorationMode and Rules.DowntiemMode, which I like, since they have different logic.

Reply:
That sounds perfect to me